containerfiles/shellai.Containerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubi9/ubi-minimal:latest
+
+# Default python version
+ARG PYTHON_VERSION=python3.12
+
+# todo: this is overriden by the image ubi9/python-311, we hard coded WORKDIR below to /app-root
+# makesure the default value of rag content is set according to APP_ROOT and then update the operator.
+ARG APP_ROOT=/app-root
+
+RUN microdnf install -y --nodocs --setopt=keepcache=0 --setopt=tsflags=nodocs \
+    ${PYTHON_VERSION} ${PYTHON_VERSION}-devel ${PYTHON_VERSION}-pip
+
+# PYTHONDONTWRITEBYTECODE 1 : disable the generation of .pyc
+# PYTHONUNBUFFERED 1 : force the stdout and stderr streams to be unbuffered
+# PYTHONCOERCECLOCALE 0, PYTHONUTF8 1 : skip legacy locales and use UTF-8 mode
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    PYTHONCOERCECLOCALE=0 \
+    PYTHONUTF8=1 \
+    PYTHONIOENCODING=UTF-8 \
+    LANG=en_US.UTF-8 \
+    PIP_NO_CACHE_DIR=off
```

Comment:
These values can be set using multiple instructions and the end result is the same. I find this easier to read, but that's just me.
```suggestion
ENV PYTHONDONTWRITEBYTECODE=1
ENV PYTHONUNBUFFERED=1
ENV PYTHONCOERCECLOCALE=0
ENV PYTHONUTF8=1
ENV PYTHONIOENCODING=UTF-8
ENV LANG=en_US.UTF-8
ENV PIP_NO_CACHE_DIR=off
```